### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Screenreader : Several texts that visually appear an heading elements are not announced as heading to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3505]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG1.3.1;#WCAG2.4.6;#MAC;#A11ySev2;#Benchmark;#Screenreader;#VoiceOver;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Navigate to View message button and click on it then start screen reader.
7. Navigate to the "View Message" text and observe if it is announced as heading or not.

### Actual:
When screen reader users navigate to the texts such as "View Message" are announced as a plain text but visually it appears to be a heading element.

This issue is observed in the most of the scenarios in the application.

### Expected:
Several texts such as "View Message" should be announced as a heading element with level 1 for screen reader users when they navigate to these texts.

### User Impact:
If the screen reader users are not provided the same information as per the visual appearance then, they might miss out on the information conveyed through it. Additionally here if the heading elements of the content on the screen are not proper then the screen reader users will not understand the structure of the content.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/Ecmszjr3Y_ZHi7nNt6InstkB9KiTZbVlFls0oZf5CDnmxQ?e=NuGA6D

**Comment by MRayermannMSFT** [comment]
Fix for this was merged:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/21112697/95634069-e2d7ac00-0a3d-11eb-87d7-7f7b54335c66.png">

**Comment by MayuriNaik** [comment]
Verified the bug in application version: 1.16.0, ios: 10.15.7. The bug is fixed and hence closing the bug.
<img width="899" alt="3505_Closed" src="https://user-images.githubusercontent.com/52036772/96449415-6c535f00-1232-11eb-99a1-dd1ea1ed2a09.png">

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #Closed
